test/tests/interfaces/pbs_partition.py

```diff
@@ -46,10 +46,9 @@ class TestPartition(TestInterfaces):
     host_name = socket.gethostname()
 
     def partition_attr(self, mgr_cmd="MGR_CMD_SET",
-                       obj_name="QUEUE", q_type="execution",
-                       q_name="Q1", enable="True",
-                       start="True", n_name=host_name,
-                       partition="P1", user=ROOT_USER):
+                       obj_name="QUEUE", q_type=None,
+                       name="Q1", enable="True",
+                       start="True", partition="P1", user=ROOT_USER):
         """
         Common function to set partition attribute/s to node/queue object
         :param mgr_cmd: qmgr "MGR_CMD_SET/MGR_CMD_UNSET/MGR_CMD_CREATE" cmd,
```

Comment:
Just a suggestion: Instead of making this a string, how about using the PTL defined symbols that expect() uses (e.g., MGR_CMD_SET)?  That way you're not doing string compares.  It'll be marginally faster.